course/content.py

```diff
@@ -224,8 +224,7 @@ def get_repo_blob(repo, full_name, commit_sha, allow_tree=True):
 
     dul_repo, full_name = get_true_repo_and_path(repo, full_name)
 
-    from os.path import split
-    names = split(full_name)
+    names = os.path.normpath(full_name).split(os.sep)
```

Comment:
The fix is here.

Reply:
Why is `str.split` better than `os.path.split`?

Reply:
`os.path.split` result in a 2-tuple, while is not appropriate where the depth of the file path is more than 2.